src/components/SearchInput/index.jsx

```diff
@@ -217,19 +254,26 @@ function SearchInput() {
           {autoCompletions.map((autoCompletion, index) => (
             <div
               key={autoCompletion}
-              className={`flex items-center w-full h-10 py-2 text-xl rounded-md ${Number(index) === selectedItemIndex ? "bg-secondary-hover" : ""}`}
-              onMouseEnter={handleMouseHover}
-              onMouseLeave={handleMouseHover}
+              className={`flex items-center w-full py-3 text-xl rounded-md ${Number(index) === selectedItemIndex ? "bg-secondary-hover" : ""} ${index < referenceIndex || userInput === "" ? "text-violet-600" : ""}`}
+              onMouseEnter={handleMouseEnter}
+              onMouseLeave={handleMouseLeave}
               onClick={handleOptionClick}
               index={index}
               role="presentation"
             >
-              {isLoggedIn ? (
+              {isLoggedIn && index < referenceIndex ? (
                 <ClockIcon className="h-6 mx-2" />
               ) : (
                 <MagnifyingGlassIcon className="h-6 mx-2" />
               )}
               {autoCompletion}
+              <span
+                className={`${(index < referenceIndex || userInput === "") && index === selectedItemIndex ? "block" : "hidden"} text-gray-500 ml-auto mr-4 hover:underline`}
```

Comment:
css 속성 순서 수정이 필요해 보입니다!
```suggestion
                className={`${(index < referenceIndex || userInput === "") && index === selectedItemIndex ? "block" : "hidden"} ml-auto mr-4 text-gray-500 hover:underline`}
```

Reply:
꼼꼼하지 못했네요! 수정하겠습니다!